box/info.go

```diff
@@ -0,0 +1,48 @@
+package box
+
+import "github.com/tarantool/go-tarantool/v2"
+
+// ClusterInfo represents information about the cluster.
+// It contains the unique identifier (UUID) of the cluster.
+type ClusterInfo struct {
+	UUID string `msgpack:"uuid"`
+}
+
+// Info represents detailed information about the Tarantool instance.
+// It includes version, node ID, read-only status, process ID, cluster information, and more.
+type Info struct {
+	// The Version of the Tarantool instance.
+	Version string `msgpack:"version"`
+	// The node ID (nullable).
+	ID *int `msgpack:"id"`
+	// Read-only (RO) status of the instance.
+	RO bool `msgpack:"ro"`
+	// UUID - Unique identifier of the instance.
+	UUID string `msgpack:"uuid"`
+	// Process ID of the instance.
+	PID int `msgpack:"pid"`
+	// Status - Current status of the instance (e.g., running, unconfigured).
+	Status string `msgpack:"status"`
+	// LSN - Log sequence number of the instance.
+	LSN uint64 `msgpack:"lsn"`
+	// Cluster information, including cluster UUID.
+	Cluster ClusterInfo `msgpack:"cluster"`
+}
+
+// Info retrieves the current information of the Tarantool instance.
+// It calls the "box.info" function and parses the result into the Info structure.
+func (b *box) Info() (Info, error) {
```

Comment:
The problem with this approach is that it cannot be extended. As example:

1. How to use a context with the method? How to cancel the request?
2. How to make an async request?

It's better to implement a custom request type, as it is already done in the `crud` and the `arrow` subpackages. It is possible to implement a custom response type for a request too:

https://github.com/tarantool/go-tarantool/blob/8cf8673dacf5ca12e55bc809ad4ab147bde93613/example_test.go#L577